### PR TITLE
feature2-v1

### PR DESCRIPTION
**Pull request by tomsonxu**
feature 2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 
+feature 2 line 1  
+
+
 line 1
 
 feature 1 line 1
 feature 1 line 2
 feature 1 line 3
 feature 1 line 4
```
Comment (cityutomsonxu) [inline README.md]: delete these lines

File: README.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 # test-pull-request
 
+feature 2 line 1  
+
+
```
Comment (cityutomsonxu) [inline README.md]: remove these 2 empty lines

**Review by cityutomsonxu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tomsonxu) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (tomsonxu) [reply README.md]: done

**Review by cityutomsonxu — APPROVED** [review]

**Pull request merged**